css/css-display/display-contents-accessibility.html

```diff
@@ -0,0 +1,321 @@
+    <div role="main" style="display: contents;" data-expectedrole="main" data-testname="div with role main and display:contents has main role" class="ex">x</div>
+    <div role="contentinfo" style="display: contents;" data-expectedrole="contentinfo" data-testname="div with role contentinfo and display:contents has contentinfo role" class="ex">x</div>
+    <div role="region" aria-label="label" style="display: contents;" data-expectedrole="region" data-testname="div with role region, aria-label and display:contents has region role" class="ex">x</div>
+    <div role="form" aria-label="label" style="display: contents;" data-expectedrole="form" data-testname="div with role form, aria-label and display:contents has form role" class="ex">x</div>
+    <div role="search" style="display: contents;" data-expectedrole="search" data-testname="div with role search and display:contents has search role" class="ex">x</div>
+
+    <!-- Links -->
+    <a href="#" style="display: contents;" data-expectedrole="link" data-testname="link with display:contents has link role" class="ex">x</a>
+    <a href="#" role="link" style="display: contents;" data-expectedrole="link" data-testname="link with explicit link role and display:contents has link role" class="ex">x</a>
+    <div role="link" style="display: contents;" data-expectedrole="link" data-testname="div with link role and display:contents has link role" class="ex">x</div>
+    <div role="link" tabindex="0" style="display: contents;" data-expectedrole="link" data-testname="div with link role, tabindex=0 and display:contents has link role" class="ex">x</div>
+
+    <!-- Lists -->
+    <ul style="display: contents;" data-expectedrole="list" data-testname="ul with display:contents has list role (child li has display:contents)" class="ex">
+        <li style="display: contents;" data-expectedrole="listitem" data-testname="li as child of ul, both with display:contents, has listitem role" class="ex">x</li>
+        <li>y</li>
+    </ul>
+
+    <ul style="display: contents;" data-expectedrole="list" data-testname="ul with display:contents has list role" class="ex">
+        <li>x</li>
+        <li data-expectedrole="listitem" data-testname="li, as child of ul with display:contents, has listitem role" class="ex">y</li>
+    </ul>
+
+    <ul>
+        <li>x</li>
+        <li style="display: contents;" data-expectedrole="listitem" data-testname="li with display:contents, as child of ul, has listitem role" class="ex">y</li>
+    </ul>
+
+    <ol style="display: contents;" data-expectedrole="list" data-testname="ol with display:contents has list role (child li has display:contents)" class="ex">
+        <li style="display: contents;" data-expectedrole="listitem" data-testname="li as child of ol, both with display:contents, has listitem role" class="ex">x</li>
```

Comment:
Same as previous comment. Test case ambiguity results in Chromium failure for an unknown (possibly legitimate) reason, and WebKit passes for an unknown reason (seems like this should be a [layout list](https://twitter.com/cookiecrook/status/1337226933822603270?lang=en) and fail legitimately).

Reply:
@cookiecrook As part of https://github.com/web-platform-tests/wpt/commit/cfad82d83cfdb185796c0ab836f1fb7771b1bce5, I've added a duplicate test with explicit role="list". Do you recommend I keep both tests, or only the one with role="list"?

Reply:
Same as previous comment.

Reply:
Got it, fixed as of: fb97fe9db208360ff58b96d4b70af2be24a1e6f4.